app/Helpers/Fixometer.php

```diff
@@ -149,8 +149,13 @@ public static function userHasDeletePartyPermission($partyId, $userId = null)
             return true;
         }
 
+        $group = Party::find($partyId)->theGroup;
+
+        if (self::hasRole($user, 'Host') && self::userIsHostOfGroup($partyId, $userId)) {
```

Comment:
Shouldn't this be passed the group id (not the party id)?

Reply:
Yes, absolutely.  Fantastically this works in the test because both the party and group id are 1 because it's a clean DB.